CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Release channels have their own copy of this changelog:
 ## [1.18.0] - Unreleased
 * Changes
   * Added a github check to support `changelog` label
+  * Uses fastboot by default (#33883)
```

Comment:
would it be better to also specify the new default CLI arg and value?

Not sure if non-involved people will know "fastboot" = `--use-snapshot-archives-at-startup when-newest` since we don't use the "fastboot" name in the CLI description 😄